jaco_manipulation/BoundingBox input
string orientation_goal
---
jaco_manipulation/BoundingBox output